README.md

```diff
@@ -40,19 +40,19 @@
 
 We love the default GitHub profiles and we want to enhance them:
 
-* The GitHub profiles aren't clearly showing all repos you have contributed to since you joined
+* The GitHub profiles clearly aren't showing all the repos you have contributed to since you joined
   GitHub. We are showing them **all**, even those you don't own and those owned by organizations
   you're not in.<sup>[1](#footnote1)</sup>
-* The GitHub profiles are listing all the repos you own but they sort them only by age of the
-  latest commit. We prefer to **sort repos** by a combination of how much you
-  have contributed to them, their size, how popular they are, etc. For each user we want to see
-  first the latest greatest repos they have most contributed to.
+* The GitHub profiles currently list all the repos you own which are sorted only by age of the
```

Comment:
here also you're changing the meaning and it doesn't mean what we had in mind anymore.

Reply:
I understand, by all means, revert this change